### ISSUE TITLE
Build fails on Linux: use of unstable library feature 'lazy_cell'

### ISSUE THREAD

**Issue by Remember20240719** [EricLBuehler/mistral.rs#721]
## Minimum reproducible example

```sh
git checkout 4b8d539330014e50e7d64fb281aa868f6579053d
cargo build --release
```

## Error
```
error[E0658]: use of unstable library feature 'lazy_cell'
  --> mistralrs-core/src/sampler.rs:17:5
   |
17 | use std::sync::LazyLock;
   |     ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> mistralrs-core/src/sampler.rs:20:31
   |
20 | ...AKERS: LazyLock<Vec<String>> =
   |           ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> mistralrs-core/src/sampler.rs:21:5
   |
21 |     LazyLock::new(|| ["\n", ":", "\"", ...
   |     ^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

error[E0658]: use of unstable library feature 'lazy_cell'
  --> mistralrs-core/src/sampler.rs:21:5
   |
21 |     LazyLock::new(|| ["\n", ":", "\"", ...
   |     ^^^^^^^^^^^^^
   |
   = note: see issue #109736 <https://github.com/rust-lang/rust/issues/109736> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `mistralrs-core` (lib) due to 4 previous errors
```

## Other information

- Operating system: Linux
- cargo 1.77.1

## Latest commit or version
Commit 4b8d539330014e50e7d64fb281aa868f6579053d

**Comment by EricLBuehler** [comment]
Can you please run `rustup update` or otherwise update Rust?

**Comment by EricLBuehler** [comment]
@Remember20240719 this should be fixed now on master in #724.

**Comment by Remember20240719** [comment]
Thanks @EricLBuehler! It works without having to upgrade cargo.
